deps/agg/include/agg_pixfmt_rgba.h

```diff
@@ -78,6 +77,64 @@ template<class ColorT, class Order> struct multiplier_rgba
     }
 };
 
+//========================================================multiplier2_rgba
+template<class ColorT, class Order, class Order2 = Order> struct multiplier2_rgba
+{
```

Comment:
The purpose was to avoid the copy needed with in-place `agg::multiplier_rgba`. Constructing color incurs a copy as well, and additionally color::demultiply is in .cpp so:
  1. can't be inlined
  2. will needlessly test premultiplied_ on every pixel